client/src/components/ManageBookings/BookingCard/BookingCard.tsx

```diff
@@ -1,32 +1,44 @@
 import { Typography, Box } from '@mui/material';
 import SettingsIcon from '@mui/icons-material/Settings';
 import useStyles from './useStyles';
+import { createImageFromInitials } from '../../../helpers/makeAnImageFromName';
+import { props } from '../../../interface/manageBooking';
+import { convertDate } from '../../../helpers/APICalls/convertDateRightFormat';
+
+const BookingCard = ({ name, start, status, photo }: props): JSX.Element => {
+  const letterImage = createImageFromInitials(name);
+
+  const firstName = name.split(' ')[0];
+  const lastName = name.split(' ')[1];
```

Comment:
This is a pretty loose way of doing this - what if someone wrote "Ethan R Moffat" then it would show last name being "R"

Let's just use their full name for simplicity.

Reply:
Or we could technically just grab the last index in the array which gives this a bit more reliability